README.md

```diff
@@ -1,10 +1,15 @@
-# Terminology Infrastructure and Management Systems Group (TIMS Group)
+# Terminology Infrastructure and Management Systems Group (TIMS Group) - website
```

Comment:
## Updated `README.md`
- Reorganized / collapsed some sections
- Added instructions on what files to edit and how to deploy
- Added some links regarding deployment/settings (collapsed)
- Other reformatting